pilot/pkg/model/context.go

```diff
@@ -673,12 +673,3 @@ func (node *Proxy) GetInterceptionMode() TrafficInterceptionMode {
 
 	return InterceptionRedirect
 }
-
-// Inbound capture listener capture all port mode only supports iptables REDIRECT
-func (node *Proxy) IsInboundCaptureAllPorts() bool {
```

Comment:
It is needed by listener_builder since virtual inbound listener was double checking if split is needed. It should be removed.

Reply:
Sorry, "should be removed" -> "can not be removed"

Reply:
? WE are no longer checking for these at all. We always create a virtual inbound listener and an outbound listener. Whether or not these listeners receive traffic depends on the iptables setup.

Reply:
how big is the inbound listener? is it just the base listener that binds to the port like today or does it have all the inbound filter chains as well? Just wondering if this will cause a substantial increase in config pushed or not

Reply:
The additional listener here is just one:

```
Name: virtualInbound
Port: 15006
useOrigDst: True
FilterChains: [
  filters: [ TCP Proxy ]
```

Nothing more than that.

Reply:
:+1: sgtm

Reply:
I had the wrong impression that the function is deleted but referenced somewhere. Well, the build is successful so I am wrong.